### PR TITLE
Remove CommandsList type

### LINKED ISSUE #173: Remove CommandsList
ℹ️ This issue is part of Epic #169 

## Objective

Apply the required changes for removing the **CommandsList**.

### PR DESCRIPTION
**Pull request by EdwinGuayacan**
Fixes #173 
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update
## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by L-Zuluaga — DISMISSED** [review]

File: lib/chainweb/pact/send_request_body.ex [hunk 1]
```diff
@@ -3,11 +3,11 @@ defmodule Kadena.Chainweb.Pact.SendRequestBody do
   `SendRequestBody` struct definition.
   """
 
-  alias Kadena.Types.{Command, CommandsList, PactTransactionHash, SignaturesList}
+  alias Kadena.Types.{Command, PactTransactionHash, SignaturesList}
 
   @behaviour Kadena.Chainweb.Pact.Type
 
-  @type cmds :: CommandsList.t()
+  @type cmds :: list(command())
```
Comment (miguelnietoa) [inline lib/chainweb/pact/send_request_body.ex:11]: @EdwinGuayacan at line 10, `command()` does not exists. Place the type command before!

Comment (miguelnietoa) [reply lib/chainweb/pact/send_request_body.ex:11]: Or write directly `list(Command.t())`!

Comment (EdwinGuayacan) [reply lib/chainweb/pact/send_request_body.ex:11]: the `command()` type exists and is used in the `@spec` of the `extract_cmds_info` function, I'm going to move it up.

Comment (miguelnietoa) [reply lib/chainweb/pact/send_request_body.ex:11]: Great

**Review by miguelnietoa — APPROVED** [review]

**Pull request merged**